tests/spec/Bar.spec.js

```diff
@@ -58,15 +53,15 @@ describe('Bar', () => {
     );
 
     const rect = bar.find('rect');
-    expect(rect).to.have.length(1);
+    expect(rect).toHaveLength(1);
 
     const rectProps = rect.props();
-    expect(rectProps.className).to.include('rct-chart-bar');
-    expect(rectProps.className).to.include('rct-chart-bar-vertical');
-    expect(rectProps.x).to.equal(50 - 20 / 2);
-    expect(rectProps.y).to.equal(25);
-    expect(rectProps.height).to.equal(50);
-    expect(rectProps.width).to.equal(20);
+    expect(rectProps.className).toContain('rct-chart-bar');
+    expect(rectProps.className).toContain('rct-chart-bar-vertical');
+    expect(rectProps.x).toEqual(50 - 20 / 2);
+    expect(rectProps.y).toEqual(25);
+    expect(rectProps.height).toEqual(50);
+    expect(rectProps.width).toEqual(20);
```

Comment:
👍 It's a big improvement using jest instead of mocha in these jsdom tests. Thanks!